front/components/assistant/conversation/SidebarMenu.tsx

```diff
@@ -84,10 +84,13 @@ export function AssistantSidebarMenu({ owner }: { owner: WorkspaceType }) {
               href={`/w/${owner.sId}/assistant/new`}
               onClick={() => {
                 setSidebarOpen(false);
-                if (
-                  router.pathname === "/w/[wId]/assistant/new" &&
-                  triggerInputAnimation
-                ) {
+                const { cId } = router.query;
```

Comment:
optional suggestion: `const isNewConv = router.asPath.startswith(``/w/${owner.sId}/assistant/new``)`

Reply:
I wanted to use the same URL as the page to limit this logic exclusively to this page.